### ISSUE TITLE
[0781 PS] A11Y issue - Missing label for \<va-select\>

### ISSUE THREAD

**Issue by lisacapaccioli** [department-of-veterans-affairs/va.gov-team#107716]
See: https://github.com/department-of-veterans-affairs/va.gov-team/issues/107325#issuecomment-2806871520

### Issue: Missing label for \<va-select\>

* **Page:** [Police report location](https://staging.va.gov/disability/file-disability-claim-form-21-526ez/mental-health-form-0781/0/event-police-report?add=true)  
* **Device:** Mac  
* **Orientation:** landscape  
* **Expected severity level:** 1  
* **Expected experience standard:** User can perceive all meaningful elements  
* **Description:** The “Country” \<va-select\> doesn’t have a label programmatically associated with it. Screen reader users that tab into this field will hear “select,” but not the word “Country”  
* **How to recreate:**   
  * Turn on VoiceOver  
  * TAB to the “Country” select  
* **Recommendation:** Add “label=”Country”” to \<va-select\>

**Comment by freeheeling** [comment]
Addressed in [draft PR](https://github.com/department-of-veterans-affairs/vets-website/pull/35972)

**Comment by freeheeling** [comment]
Merged.
